dataverse-webapp/src/main/webapp/createEditDataverse.xhtml

```diff
@@ -72,148 +72,137 @@
                             </label>
                             <div class="col-sm-9 form-col-container">
-                                <div class="input-group">
-                                    <div class="input-group-addon"><span
-                                            class="small">#{systemConfig.dataverseSiteUrl}/dataverse/</span></div>
-                                    <p:inputText id="identifier" tabindex="3" styleClass="form-control"
-                                                 validator="#{CreateEditDataversePage.validateAlias}"
-                                                 value="#{CreateEditDataversePage.dataverse.alias}"/>
-                                </div>
+                                <p class="help-block dataverse-general-info-address-field">
+                                    <span class="small">#{systemConfig.dataverseSiteUrl}/dataverse/</span>
+                                </p>
+                                <p:inputText id="identifier" styleClass="form-control"
+                                                validator="#{CreateEditDataversePage.validateAlias}"
+                                                value="#{CreateEditDataversePage.dataverse.alias}"/>
                                 <p:message for="identifier" display="text"/>
                             </div>
                         </div>
-                        <div class="col-xs-6 form-group" jsf:rendered="#{CreateEditDataversePage.ownerId != null}">
-                            <label class="col-sm-3 control-label" for="hostDataverse">
-                                #{bundle.hostDataverse}
+                        <div class="col-md-6 form-group form-col-container">
+                            <label class="col-sm-3 control-label" for="dataverseCategory">
+                                #{bundle['dataverse.category']} <span
+                                    class="glyphicon glyphicon-asterisk text-danger"
+                                    title="#{bundle.requiredField}"/>
                                 <span class="glyphicon glyphicon-question-sign tooltip-icon"
-                                      data-toggle="tooltip" data-placement="auto right"
-                                      data-original-title="#{bundle['dataverse.host.title']}"></span>
+                                        data-toggle="tooltip" data-placement="auto right"
+                                        data-original-title="#{bundle['dataverse.category.title']}"></span>
                             </label>
                             <div class="col-sm-9 form-col-container">
-                                <p class="form-control-static">#{CreateEditDataversePage.dataverse.getOwner().name}</p>
-                                <h:inputHidden value="#{CreateEditDataversePage.ownerId}" id="hostDataverse"/>
+                                <p:selectOneMenu id="dataverseCategory" tabindex="5" styleClass="form-control"
```

Comment:
probably tabindex here is bad for alt+tab. I can't reach into this selectbox

Reply:
Yes, there was a leftover `tabindex` attribute that I forgot to remove. Fixed.